tests/integration/targets/mso_schema_template_migrate/tasks/main.yml

```diff
@@ -267,54 +269,58 @@
   when: version.current.version is version('2.2.4e', '!=')
   register: object_migrate
 
-- name: Migration of BD objects between templates
-  mso_schema_template_migrate:
-    <<: *mso_info
-    schema: '{{ mso_schema | default("ansible_test") }}_2'
-    template: Template 1
-    target_schema: '{{ mso_schema | default("ansible_test") }}_2'
-    target_template: Template 2
-    bds:
-    - ansible_test_2
-    state: present
-  when: version.current.version is version('2.2.4e', '!=')
-  register: bd_migrate
+- name: Execute tasks only for MSO version < 4.0 # TODO not able to execute multiple migrations. Deploy is needed due to validation in NDO4.0.
```

Comment:
With the new deploy would this be able to be done now?

Reply:
that should be the case, will add those